src/ray/protobuf/common.proto

```diff
@@ -195,6 +195,7 @@ enum ErrorType {
   // Indicates that the object has been placed in plasma. This error shouldn't
   // ever be exposed to user code; it is only used internally to indicate the
   // result of a direct call has been placed in plasma.
+  // IMPORTANT: Keep the enum index "4" in sync with ray_object.cc.
```

Comment:
and there should be < 100 error types. I think realistically, it won't happen, but nice to mention..